run-tests.pl

```diff
@@ -34,17 +35,22 @@
       : undef;
 });
 
+my $plugin_dir = $ENV{"SYTEST_PLUGINS"} || "plugins"; # Read plugin dir from env var SYTEST_PLUGINS or fallback to plugins
+my @plugins = grep { -d } glob(catdir($plugin_dir, "*"));
```

Comment:
Needs some rework, because bootstrap.sh will download into `plugins/<owner>/<plugin>` 
glob probably won't list all dirs 2 layers deep? I do not write perl fluently.